kernel/include/paging.h

```diff
@@ -0,0 +1,53 @@
+//https://wiki.osdev.org/Paging         x86 2 layer paging
+//https://os.phil-opp.com/page-tables/  x64 4 layer paging
+//P4 P3 P2 P1 represent the different pagetable layers
+
+
+//uefi pagetables are placed in a nonreadable section by default
+
+#pragma once
+#include <stdint.h>
+#include <stdbool.h>
+
+//temporary solution before memory allocation is done
+extern uint64_t page_zone_stack[] __attribute__((aligned(4096)));
+extern uint64_t* page_zone_pointer;
+
+#define f_paging_present    (1L << 0)
```

Comment:
Defines should be all caps. Example: PAGING_PRESENT

Do all of them need to be in the header or are some only important to the implementation?